target/linux/realtek/files-6.12/drivers/net/dsa/rtl83xx/rtl931x.c

```diff
@@ -675,11 +676,22 @@ static void rtl931x_write_l2_entry_using_hash(u32 hash, u32 pos, struct rtl838x_
 	u32 r[4];
 	struct table_reg *q = rtl_table_get(RTL9310_TBL_0, 0);
 	u32 idx = (0 << 14) | (hash << 2) | pos; /* Access SRAM, with hash and at pos in bucket */
+	bool hash_algo;
```

Comment:
Please use something like "int hash_algo_id" to make clear that this is sume kind of numbering. Both cases will use some kind of algorithm.

Reply:
Done, please check the change.